stacker_blueprints/generic.py

```diff
@@ -0,0 +1,77 @@
+""" Load dependencies """
+from troposphere import (
+    Ref, Output
+)
+
+from stacker.blueprints.base import Blueprint
+
+
+class GenericResourceCreator(Blueprint):
+    """ Generic Blueprint for creating a resource """
+    def add_cfn_description(self):
+        """ Boilerplate for CFN Template """
+        template = self.template
+        template.add_version('2010-09-09')
+        template.add_description('CGM Generic Resource Creator - 1.0.0')
+
+    """
+
+    *** NOTE ***    Template Version Reminder
+
+    Make Sure you bump up the template version number above if submitting
+    updates to the repo. This is the only way we can tell which version of
+    a template is in place on a running resouce.
+
+    """
+
+    VARIABLES = {
+        'Class':
+            {'type': str,
+             'description': 'The troposphere class to create'},
+        'Output':
+            {'type': str,
+             'description': 'The output to create'},
+        'Properties':
+            {'type': dict,
+             'description': 'The list of propertie to use for the troposphere'
+                            + ' class'},
```

Comment:
You don't need the `+` here for the continuation.  Also, small nit, but can you include the space on the previous line, rather than leading on this line?  Thanks!